builtin/staker/validation/validation.go

```diff
@@ -122,6 +122,15 @@ func (v *Validation) CurrentIteration() uint32 {
 func (v *Validation) Renew() *delta.Renewal {
 	newLockedVET := big.NewInt(0)
 
+	if v.QueuedVET == nil {
```

Comment:
Would it make sense to have a `New()` function where all these values are initialized? Or they might be null during their lifecycle?